Source/WebInspectorUI/UserInterface/Views/LocalResourceOverridePopover.js

```diff
@@ -362,8 +362,10 @@ WI.LocalResourceOverridePopover = class LocalResourceOverridePopover extends WI.
         };
 
         let toggleHeadersDataGridVisibility = (force) => {
-            this._headersDataGrid.element.hidden = force !== undefined ? force : !this._headersDataGrid.hasChildren;
-            this._headersDataGrid.updateLayoutIfNeeded();
+            let hidden = force !== undefined ? force : !this._headersDataGrid.hasChildren;
```

Comment:
Nit: Do we have to interpret `null` specially here, or could we just `let hidden = force ?? !this._headersDataGrid.hasChildren;`?

Reply:
ooo yeah `force ??` would work!